darwin-configuration.nix

```diff
@@ -16,6 +16,9 @@
   nix.settings.sandbox = "relaxed";
   nix.settings.extra-platforms = [ "x86_64-darwin" ];
 
+  nix.nrBuildUsers = config.nix.settings.max-jobs * 2;
```

Comment:
x2 for extra headroom?

Reply:
just copied over since that's what aarch64-build-box is using, not sure if we need this

Reply:
If Graham did it, it's probably good, assuming it even evals.